db/migrate/20201012072557_create_users_gdpr_delete_requests.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsersGdprDeleteRequests < ActiveRecord::Migration[6.0]
+  def change
+    create_table :users_gdpr_delete_requests do |t|
+      t.integer :user_id
+      t.string :email
```

Comment:
Should the email and/or username be mandatory, i.e. `null: false`? 

Also: should they have a simple boolean status column, e.g. `done`? The reason I'm asking is that we may want a uniquess validation across all these attributes so that a user can only have one active open request, but theoretically more in the future (sign up, trigger GDPR delete, sign up again later, trigger GDPR delete again).

Reply:
Is it an issue that we are now storing the user's email on this GdprDeleteRequest when the point is to delete all of the user's GDPR data which I assume includes their email?

Reply:
Currently, the record is deleted when an admin confirms that the data from 3-rd party services was deleted. I thought that we shouldn't keep emails as the data should be deleted :thinking:

Reply:
I think I might be confused as to the flow here. So is this what the flow looks like?
1. User deletes themselves 
2. User delete triggers `GdprDeleteRequest` to be created, is that right???
3. GDPR data is deleted by admin
4. Admin deletes `GdprDeleteRequest`

Reply:
Yes, that's how I understand the flow too @mstruve. The confirm button actually deletes the entry in `GdprDeleteRequest` via a `DELETE` request.

Reply:
@mstruve That's right, after admin confirms, the record is destroyed, so we won't be storing email anymore.